### PR TITLE
F#pd 33 split mst public private

### PR DESCRIPTION
**Pull request by carebare47**
## Proposed changes

Adding arrow publisher, launch files, visualiser configs from sr_mst

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

**Review by rnzenha-s — APPROVED** [review]

File: sr_tactile_sensors/launch/sr_mst_hand_plotjuggler_visualiser.launch [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ more details.
+ You should have received a copy of the GNU General Public License along
+ with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<launch>
+  <!-- Hand and sensor configurations-->
+  <arg name="hand_id" default="rh"/>
+
+  <node name="$(arg hand_id)_MST_plotjuggler_visualiser" pkg="plotjuggler" type="plotjuggler" args="--layout $(find sr_tactile_sensors)/config/hand_plotjuggler_config_$(arg hand_id).xml"/>
```
Comment (toliver) [inline sr_tactile_sensors/launch/sr_mst_hand_plotjuggler_visualiser.launch:18]: We probably should add `plotjuggler` as a `<run_depend>` to the package.xml

**Review by toliver — CHANGES_REQUESTED** [review]
I assume we only need to review the moving, not the actual code.

File: sr_tactile_sensors/launch/sr_mst_hand_plotjuggler_visualiser.launch (hunk 1 above)
Comment (carebare47) [reply sr_tactile_sensors/launch/sr_mst_hand_plotjuggler_visualiser.launch:18]: good point. Shouldn't CI have caught that?

Comment (carebare47) [reply sr_tactile_sensors/launch/sr_mst_hand_plotjuggler_visualiser.launch:18]: c70d4a9b409ff564d10980c78b32fdacc9e94db0

File: sr_tactile_sensors/package.xml [hunk 2]
```diff
@@ -43,6 +43,10 @@
     <run_depend>sr_robot_msgs</run_depend>
     <run_depend>visualization_msgs</run_depend>
     <run_depend>gazebo_msgs</run_depend>
-    <run_depend>gazebo_ros</run_depend>
+    <run_depend>gazebo_ros</run_depend>    
+
+  <run_depend>plotjuggler</run_depend>
```
Comment (toliver) [inline sr_tactile_sensors/package.xml]: Bad indentation

**Review by toliver — CHANGES_REQUESTED** [review]

File: sr_tactile_sensors/package.xml (hunk 2 above)
Comment (carebare47) [reply sr_tactile_sensors/package.xml]: 9d85183afd9919c309266d7be935455be32970f5

**Review by toliver — APPROVED** [review]

**Pull request merged**